src/modules/api/pipeline-task/pipelines/utils.py

```diff
@@ -36,131 +26,89 @@ def load_json_from_request(request):
     raise APIBadRequestError('Failed to parse request body as valid JSON') from ex
 
 
-def log_ssl_backoff(details):
-  logger.warn(f'[TASK {details["args"][0].get("id", "no-id")}] Encountered SSLEOFError trying to start job. Trying again in {details["wait"]:00.1f}s...')
-
-def log_ssl_giveup(details):
-  logger.warn(f'[TASK {details["args"][0].get("id", "no-id")}] Encountered SSLEOFError trying to start job. Giving up. Total time elapsed: {details["elapsed"]:00.1f}s.')
-
-
 
 def get_task_handler(queue_name, *args, **kwargs):
```

Comment:
Now returns an object of the appropriate `JobPipeline` subclass, instead of a `TaskHandler`